src/main/java/mate/academy/exceptions/EntityNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.exceptions;
+
+public class EntityNotFoundException extends RuntimeException {
+    public EntityNotFoundException(String message, Throwable cause) {

```

Comment:
Consider adding another constructor that only takes a message as a parameter. This provides more flexibility when throwing exceptions.

Reply:
ok, done